### ISSUE TITLE
Dashboard fails to load after changing TRACKER_SCRIPT_NAME and COLLECT_API_ENDPOINT

### ISSUE THREAD

**Issue by tomaae** [umami-software/umami#1858]
Hello,
After changing TRACKER_SCRIPT_NAME  and COLLECT_API_ENDPOINT, dashboard will no longer load for new sessions. Only old open session remained workign in chrome for some reason.
Tracking itself works, its just dashboard.
When I remove both parameter and rebuild it, it starts to work.

Version: 1.40

.env
```
TRACKER_SCRIPT_NAME=a
COLLECT_API_ENDPOINT=/api/c
```

http response
```
POST 	https://mydomain/_next/static/chunks/api/c
{
	"Status": "400Bad Request",
	"Version": "HTTP/2",
	"Transferred": "400 B (15 B size)",
	"Referrer Policy": "strict-origin-when-cross-origin"
}
```

pm2 error log, but they dont seem to concide with dashboard load time.
```
Error: Invalid request
    at getSession (/srv/umami/.next/server/chunks/1649.js:273:15)
    at /srv/umami/.next/server/chunks/1649.js:114:86
    at /srv/umami/node_modules/next-basics/dist/cjs/index.js:1:66323
    at new Promise (<anonymous>)
    at /srv/umami/node_modules/next-basics/dist/cjs/index.js:1:66296
    at __WEBPACK_DEFAULT_EXPORT__ (/srv/umami/.next/server/pages/api/collect.js:209:79)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.apiResolver (/srv/umami/node_modules/next/dist/server/api-utils/node.js:366:9)
    at async NextNodeServer.runApi (/srv/umami/node_modules/next/dist/server/next-server.js:481:9)
    at async Object.fn (/srv/umami/node_modules/next/dist/server/next-server.js:735:37)
TypeError: Cannot read properties of null (reading 'id')
    at relationalQuery (/srv/umami/.next/server/chunks/9447.js:1712:65)
    at Object.prisma (/srv/umami/.next/server/chunks/9447.js:1708:69)
    at runQuery (/srv/umami/.next/server/chunks/9447.js:296:31)
    at savePageView (/srv/umami/.next/server/chunks/9447.js:1707:70)
    at __WEBPACK_DEFAULT_EXPORT__ (/srv/umami/.next/server/pages/api/collect.js:218:78)
    at async Object.apiResolver (/srv/umami/node_modules/next/dist/server/api-utils/node.js:366:9)
    at async NextNodeServer.runApi (/srv/umami/node_modules/next/dist/server/next-server.js:481:9)
    at async Object.fn (/srv/umami/node_modules/next/dist/server/next-server.js:735:37)
    at async Router.execute (/srv/umami/node_modules/next/dist/server/router.js:247:36)
    at async NextNodeServer.run (/srv/umami/node_modules/next/dist/server/base-server.js:347:29)
```

**Comment by franciscao633** [comment]
Can you try changing the TRACKER_SCRIPT_NAME to something other then `a`. #1769 . Let me know if this works for you.

**Comment by tomaae** [comment]
Changed as following and rebuild:
```
TRACKER_SCRIPT_NAME=st
COLLECT_API_ENDPOINT=/api/co
```

Login screen showed up. But after logging in, page just keeps reloading with same issue:
```
Request URL: https://mydomain/_next/static/N_e03hJRyA4a7bjAa_NaP/api/co
{
	"Status": "400Bad Request",
	"Version": "HTTP/2",
	"Transferred": "400 B (15 B size)",
	"Referrer Policy": "strict-origin-when-cross-origin"
}
```

**Comment by tomaae** [comment]
As much as we like umami, we have to drop it as its single thread nature does not make it suitable for larger traffic sites and also dashboard crashes due to amount of data.
I will leave this issue open so it can be fixed for others, but I'm no longer able to test it myself. Maybe we try again sometimes in future once umami can handle more traffic.

**Comment by mikecao** [comment]
This is resolved in v2.

**Comment by tomaae** [comment]
I gave it try when 2.0 was released actually.
TRACKER_SCRIPT_NAME does nothing. COLLECT_API_ENDPOINT gives error too, but I dont remember if it was on tracked site or dashboard. so I had to removed both to make it work.
realtime visitor count was broken, showing always 1, but in dashboard it was ok.
after about 6 hours dashboard was taking 40 seconds to load data. so it got much worse at handling higher traffic
